src/build123d/build_common.py

```diff
@@ -1176,6 +1176,77 @@ def localize(cls, *points: VectorLike) -> Union[list[Vector], Vector]:
         return result
 
 
+def solids(select: Select = Select.ALL) -> ShapeList[Solid]:
+    """
+    Return Solids for the current builder context
+
+    Return either all or the solids created during the last operation.
+
+    Args:
+        select (Select, optional): Solid selector. Defaults to Select.ALL.
+
+    Returns:
+        ShapeList[Solid]: Solids extracted
+    """
+    return Builder._get_context("solids").solids(select)
```

Comment:
When used in Algebra, there is no Builder context so this fails:
```py
>>> from build123d import *
>>> square = Rectangle(10,10)
>>> corners = vertices()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/roger-maitland/Documents/build123d/src/build123d/build_common.py", line 1200, in vertices
    return Builder._get_context("vertices").vertices(select)
AttributeError: 'NoneType' object has no attribute 'vertices'
```
The context should be checked for None and if there isn't one available a RunTime exception should be generated to tell the user that this operation only applies to Builder mode (i.e. an Builder context must be in scope).